THIRTEEN_secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: app-secret
+data:
+  DB_Host: bXlzcWw=
+  DB_user: cm9vdA==
+  DB_Password: cGFzd3Jk
```

Comment:
It's not critical for this particular case, however, always keep in mind that secrets should not be stored in git